jni/Android.mk

```diff
@@ -82,9 +86,23 @@ LOCAL_SRC_FILES := $(OPENBLAS_ROOT)/lib/libopenblas.a
 LOCAL_EXPORT_C_INCLUDES := $(OPENBLAS_ROOT)/include
 LOCAL_EXPORT_CFLAGS += -DUSE_BLAS=1
 
+include $(CLEAR_VARS)
+
 include $(PREBUILT_STATIC_LIBRARY)
 endif #ENABLE_BLAS
 
+## prepare ml common api if nothing present
+ifndef ML_API_COMMON_ROOT
+ifneq ($(MAKECMDGOALS),clean)
+
+ML_API_COMMON_ROOT := $(NDK_INCLUDES_OUT)/ml_api_common
+$(info $(shell ($(NNTRAINER_JNI_ROOT)/prepare_ml-api-common.sh $(ML_API_COMMON_ROOT))))
```

Comment:
Did pkgconfig strategy not work?

Reply:
There is no way to enforce the package to be available when building for jni.

Reply:
True that